### DISCUSSION TITLE
Please update to RNS 0.3.11, two important bugs fixed

### DISCUSSION THREAD

**Discussion by markqvist** [markqvist/Reticulum#80] [Announcements]
This is just a quick note to let you all know that in version 0.3.9, two bugs were introduced due to the migration to the 128-bit address space. These are both fixed in the newly released v0.3.11, and I highly recommend updating as soon as possible.

**Comment by kc1awv** [comment]
I think I just realized what happened with my addresses while reading this. I noticed that there seems to be no announces coming off of the test net though - is the test net waiting for an upgrade?

**Reply by markqvist** [reply]
Did you already upgrade to 0.3.9+? Since the address space upgrade, I changed the TCP and I2P entry points to avoid confusion with 80-bit nodes trying to connect to the new 128-bit testnet. If you didn't update your Reticulum config files, take a look at the new settings below:

```
# TCP/IP interface to the Dublin Hub
  [[RNS Testnet Dublin]]
    type = TCPClientInterface
    enabled = yes
    target_host = dublin.connect.reticulum.network
    target_port = 4965

# TCP/IP interface to the Frankfurt Hub
  [[RNS Testnet Frankfurt]]
    type = TCPClientInterface
    enabled = yes
    target_host = frankfurt.connect.reticulum.network
    target_port = 5377

# Interface to I2P Hub A
  [[RNS Testnet I2P Hub A]]
    type = I2PInterface
    enabled = yes
    peers = mrwqlsioq4hoo2lmeeud7dkfscnm7yxak7dmiyvsrnpfag3z5tsq.b32.i2p

# Interface to I2P Hub B
  [[RNS Testnet I2P Hub B]]
    type = I2PInterface
    enabled = yes
    peers = iwoqtz22dsr73aemwpw7guocplsjjoamyl7sogj33qtcd6ds4mza.b32.i2p
```

**Reply by kc1awv** [reply]
> I changed the TCP and I2P entry points to avoid confusion with 80-bit nodes trying to connect to the new 128-bit testnet.

Aha! That'll explain it.

I probably should pay better attention to what's going on. Ok, I switched my 'persistent' node over to the new addresses and everything's coming through again. Thanks for the heads up!

**Reply by markqvist** [reply]
No worries, I tried getting all the info visible, but apparently it wasn't a complete success ;) Good to hear it was just that!
